src/components/App/App.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
-import SearchBar from './SearchBar';
-import SelectedPokemon from './SelectedPokemon';
-import PokemonList from './PokemonList';
-import { getData } from '../actions';
+import SearchBar from '../SearchBar/SearchBar';
```

Comment:
unikamy default exportów tak jak piasłem w mailu,

czyli zamiast 
  `export deafult sth` 
robisz
   `export const aName = sth`

i importujesz tak
  `import { aName } from 'path/to/file'`